cdap-ui/app/directives/my-post-run-action-wizard/my-post-run-action-wizard.js

```diff
@@ -47,6 +48,27 @@ angular.module(PKG.name + '.commons')
           $scope.store = rStore;
           $scope.mode = rMode;
           $scope.action = rAction;
+          $scope.myPipelineApi = rStore.myPipelineApi;
+          $scope.validating = false;
+          $scope.showValidateButton = function() {
```

Comment:
I think you'll have to check the mode as well, to not show it when it's on `view`